go.mod

```diff
@@ -22,23 +22,21 @@ module github.com/apache/yunikorn-scheduler-interface
 go 1.20
 
 require (
-	github.com/golang/protobuf v1.2.0
-	golang.org/x/net v0.8.0
-	google.golang.org/grpc v1.23.1
-	google.golang.org/protobuf v1.26.0-rc.1
+	github.com/golang/protobuf v1.5.3
+	google.golang.org/grpc v1.56.0
+	google.golang.org/protobuf v1.30.0
 )
 
 require (
-	golang.org/x/sys v0.6.0 // indirect
-	golang.org/x/text v0.8.0 // indirect
-	google.golang.org/genproto v0.0.0-20180817151627-c66870c02cf8 // indirect
+	golang.org/x/net v0.9.0 // indirect
+	golang.org/x/sys v0.7.0 // indirect
+	golang.org/x/text v0.9.0 // indirect
+	google.golang.org/genproto v0.0.0-20230410155749-daa745c078e1 // indirect
 )
 
 replace (
```

Comment:
Can we also update these replace values?
Keep them insync with the ones above etc